server/commands.go

```diff
@@ -274,6 +276,20 @@ func NewServerCommands(defaultCfg *sconfig.Config) []*cli.Command {
 				}
 				if c.String(common.FlagDBPath) == "" {
 					opts = append(opts, WithPersistenceDisabled())
+
+					if clusterCfg, err := cliconfig.NewConfig("temporalio", "cluster"); err == nil {
```

Comment:
Maybe call this:

```suggestion
					if clusterCfg, err := cliconfig.NewConfig("temporalio", "server"); err == nil {
```

Reply:
So we can add more server configs there if we want.

Reply:
Oh and it could be better to have the same structure as the server config.

Reply:
i'll look into reusing server's config structure

Reply:
On second thought, having a separate file just for this is better, now that you changed the name I'm okay with the PR as is.